.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.py]
+charset = utf-8
+indent_size = 4
+
+
+[{*.json,*.toml,*.yaml}]
+indent_size = 2
```

Comment:
What's the point of this file? Just a formatting script?

Reply:
"...helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs." ([source](https://editorconfig.org/))